Harvest.Web/ClientApp/src/Tickets/TicketCreate.tsx

```diff
@@ -42,8 +45,30 @@ export const TicketCreate = () => {
   if (project === undefined) {
     return <div>Loading...</div>;
   }
+
+  
+  const checkTicketValidity = async (inputs: any) => {
+    try {
+      await ticketSchema.validate(inputs, { abortEarly: false });
+    } catch (err) {
+      if (err instanceof ValidationError) {
+        return err.errors;
+      }
+    }
+  };
+
   const create = async () => {
-    // TODO: validation
+
+    const ticketErrors = await checkTicketValidity(ticket);
+
+    if (ticketErrors) {
+      if (ticketErrors.length > 0) {
```

Comment:
I believe you can do null conditionals just like C#: `if (ticketErrors?.length > 0)`